README.md

```diff
@@ -6,17 +6,18 @@
 
 **Welcome to Maintainer Week!**
 
-Open Source runs the world. But who holds the reins? Who is there, day by day, to coordinate keeping it running? Open Source project maintainers make the software we use possible. But project maintenance is intense work - and it doesn’t happen in a vacuum.
+Open Source runs the world. But runs open source? Who is there, day by day, to coordinate keeping it running? Open Source project maintainers make the software we use possible. But project maintenance is intense work - and it doesn’t happen in a vacuum.
```

Comment:
"who runs open source." Also should we be consistent through out with "open source" vs "Open Source"?

Also suggest removing "coordinate" in favor of "to keep it running"